Objects/longobject.c

```diff
@@ -2193,9 +2193,8 @@ unsigned char _PyLong_DigitValue[256] = {
     37, 37, 37, 37, 37, 37, 37, 37, 37, 37, 37, 37, 37, 37, 37, 37,
 };
 
-/* *str points to the first digit in a string of base `base` digits.  base
- * is a power of 2 (2, 4, 8, 16, or 32).  *str is set to point to the first
- * non-digit (which may be *str!).  A normalized int is returned.
+/* `start` and `end` point to the start end of a string of base `base` digits.
+ * base is a power of 2 (2, 4, 8, 16, or 32). An unnormalized int is returned.
```

Comment:
Please could you add a description of the new `digits` parameter? (It would be good to clarify that it ignores underscores, so is not the same as `end - start`.)